src/components/display-value.tsx

```diff
@@ -22,6 +23,39 @@ const DisplayValue = ({
   className,
   styles,
 }: DisplayValueProps): JSX.Element => {
+  /**
+   * see if there's a data-chaise-tooltip in the displayed value, and turn them into proper tooltips.
+   *
+   * NOTE:
+   * I'm using bootstrap.js for this feature. this has added around 30KB to our bundles. I couldn't find a way to do this
+   * directly with react-bootstrap. but there might be a way and we should investigate later
+   */
+  const spanRef = useRef<HTMLSpanElement | null>(null);
+  useEffect(() => {
+    if (!spanRef.current) return;
+
+    const tooltipTriggerList = document.querySelectorAll('[data-chaise-tooltip]');
+    if (tooltipTriggerList && tooltipTriggerList.length > 0) {
+      tooltipTriggerList.forEach((el) => {
+        const title = el.getAttribute('data-chaise-tooltip');
+        const placement = el.getAttribute('data-chaise-tooltip-placement') || 'bottom';
+        const noIcon = el.hasAttribute('data-chaise-tooltip-no-icon');
+        if (!title) return;
+        if (!noIcon) {
+          // adding space between content and the icon is how we're making sure spacing between the two is correct.
+          // should we come up with a better solution instead?
+          el.innerHTML = el.innerHTML + ' ';
+          el.classList.add('chaise-icon-for-tooltip');
+        }
+        new Tooltip(el, {
+          title,
+          // @ts-ignore ts doesn't understand that we're actually sanitizing the value.
+          placement: ['auto', 'top', 'bottom', 'left', 'right'].indexOf(placement) !== -1 ? placement : 'bottom'
+        })
+      });
+    }
+  });
```

Comment:
Should the dependency array for this be `undefined`? My understanding is that an `undefined` dependency array will fire useEffect on each render but we only need to run this once when the `spanRef` is attached to the page